django/test/testcases.py

```diff
@@ -1089,6 +1094,7 @@ class TransactionTestCase(SimpleTestCase):
 
     # Subclasses can enable only a subset of apps for faster tests
     available_apps = None
+    _available_apps_calls_balanced = True
```

Comment:
I would have suggested to use an int here because of the name ("balanced"), where the set increases and the unset decreases. Balanced would be 0.
If we want to keep the boolean, shall we name this "overriden"?
```suggestion
    _available_apps_overriden = False
```

Reply:
Great idea, updated to use an int.